src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,148 @@
+        1. Toggle the switch called **Find out if my company qualifies for an Enterprise Free license**. *By toggling this switch you are legally attesting to the fact that your company revenue meets the requirements of the license*.
+        1. Click the **Continue** button.
+        1. Select the checkbox to agree to the Terms & Conditions of the [CockroachDB Software License][csl].
+        1. Click the **Generate License Key** button. You will be redirected to the **Enterprise Licenses** page, where you can [start using the key](#set-a-license).
+
+{{site.data.alerts.callout_danger}}
+You will not be able to create more than one **Enterprise Trial** license per day. If you try, the UI will prevent you from proceeding, and the following message will be displayed:
+
+> A new Enterprise Trial license cannot be created at this time because one was recently created. Tell us more about your intended use to find out if you qualify for an Enterprise Free license, or create a new one after October 16, 2024 at 10:49 AM EDT.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+For quick local testing of Enterprise features, you can [run a single-node cluster](#single-node-clusters).
+{{site.data.alerts.end}}
+
+## Set a license
+
+{% include common/set-enterprise-license.md %}
+
+## Verify a license
+
+To verify a license, open the [built-in SQL shell]({% link {{ page.version.version }}/cockroach-sql.md %}) and use the [`SHOW CLUSTER SETTING`]({% link {{ page.version.version }}/show-cluster-setting.md %}) command to check the license key:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+>  SHOW CLUSTER SETTING enterprise.license;
+~~~
+~~~
+             enterprise.license
++-------------------------------------------+
+  crl-0-ChB1x...
+(1 row)
+~~~
+
+The license setting is also logged in the cockroach.log on the node where the command is run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+$ cat cockroach.log | grep license
+~~~
+~~~
+I171116 18:11:48.279604 1514 sql/event_log.go:102  [client=[::1]:56357,user=root,n1] Event: "set_cluster_setting", target: 0, info: {SettingName:enterprise.license Value:xxxxxxxxxxxx User:root}
+~~~
+
+## Monitor for license expiry
+
+You can monitor the time until your license expires in the following ways:
+
+1. [Prometheus]({% link {{ page.version.version }}/monitor-cockroachdb-with-prometheus.md %}): The `seconds_until_enterprise_license_expiry` metric reports the number of seconds until the license on a cluster expires. It will report `0` if there is no license, and a negative number if the license has already expired. For more information, see [Monitoring and Alerting]({% link {{ page.version.version }}/monitoring-and-alerting.md %}).
+1. [DB console]({% link {{ page.version.version }}/ui-overview.md %}): Several [license expiration messages]({% link {{ page.version.version }}/ui-overview.md %}#license-expiration-message) may be displayed, depending on the status of your cluster's license.
```

Comment:
For Enterprise Free and Trial clusters, we also notify you in the CC console under the Enterprise Licenses section. We will also send emails accordingly.

Reply:
Nice, added another bullet:

1. [CockroachDB {{ site.data.products.cloud }} Console][cloud-console]: If you have an **Enterprise Free** or **Enterprise Trial** cluster, you will see notifications in the **Enterprise Licenses** section, as well as receive notification emails to the [email address associated with your account](#obtain-a-license).

Reply:
great! the only additional comment is that emails are sent to the cloud org's admins (i.e., users with Org Admin permissions).

Reply:
modified to say the following in commit named `Emails go to those with Org Admin perms`

> ... as well as receive notification emails sent to users with **Org Admin** permissions.

PTAL!